frontend/src/app/forgotPassword/page.tsx

```diff
@@ -1,23 +1,25 @@
-import { Metadata } from "next";
-import Logo from "../components/HomeComponents/Logo/logo";
-import LandingImage from "../components/landingimage";
-import ForgotPassword from "../components/ForgotPasswordComponents/forgotpassword";
+import React from 'react';
+import { type Metadata } from 'next';
+
```

Comment:
This blank row could be removed.

Reply:
I prefer to leave a free line between logical units for a better structure of imports, if you don't mind :)